test/lib/jdk/test/lib/util/FileUtils.java

```diff
@@ -377,6 +377,21 @@ public static long getProcessHandleCount() {
         }
     }
 
+    public static void patch(Path path, int fromLine, int toLine, String to) throws IOException {
+        if(fromLine < 1 || toLine < 1) {
```

Comment:
It would be good to add a proper API doc comment, especially regarding the meaning of the parameters, and whether the line in question is included/excluded. Also RuntimeException could be replaced with IndexOutOfBoundsException.